ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -31,13 +48,16 @@ import SwiftUI
 /// View for List Cells
 struct MSFListCellView: View {
     @ObservedObject var state: MSFListCellState
-    @ObservedObject var tokens: MSFListTokens
+    @ObservedObject var tokens: MSFListCellTokens
     var hasDividers: Bool
 
-    init(state: MSFListCellState, tokens: MSFListTokens, hasDividers: Bool = false) {
+    init(state: MSFListCellState, hasDividers: Bool = false, windowProvider: FluentUIWindowProvider?) {
         self.state = state
-        self.tokens = tokens
         self.hasDividers = hasDividers
+        self.tokens = MSFListCellTokens(cellLeadingViewSize: state.leadingViewSize)
+        if let windowProvider = windowProvider {
```

Comment:
The windowProvider property is **FluentUIWindowProvider?**.
You can just set the value passed as a parameter directly. (without the if let)